fenix/app/src/main/java/org/mozilla/fenix/compose/ClickableSubstringLink.kt

```diff
@@ -32,6 +32,7 @@ import org.mozilla.fenix.theme.FirefoxTheme
 @Composable
 fun ClickableSubstringLink(
     text: String,
+    textStyle: TextStyle = FirefoxTheme.typography.caption,
```

Comment:
I think this is fine for now, 

but going forward, we should use `LinkText` in onboarding after migrating it here or use the same api so every caller is not calculating the index and add accessibility support. There's a separate bug for that, that also needs to support multiple LinkTexts in one Text.

Reply:
ok, that makes sense to me.